scripts/fbi/hate_crime/README.md

```diff
@@ -0,0 +1,43 @@
+# Importing FBI Hate Crime Data
+
+This directory imports [FBI Hate Crime Data](https://ucr.fbi.gov/hate-crime) into Data Commons. It includes data at US, state and city level.
+
+## Generating Artifacts:
+To generate `cleaned.csv`, `output.mcf` for a publication. run:
+
+```bash
+cd table<publication number>
+python3 preprocess.py
+```
+
+### Data Caveats:
+- New Jersey is missing data for year 2012 in publications 11 and 12
+- Data for a few locations of crime (publication 10) are missing for certain years
```

Comment:
Please add a note if the output.mcf is not to be used and instead the statvars definition comes from the aggregations scripts due to the change in populationType, etc.

Reply:
Done.